helpers.py

```diff
@@ -229,12 +229,37 @@ def files_changed(diff, file_set):
 
 
 core_files = {
-    "apigetpost.py", "blacklists.py", "bodyfetcher.py", "chatcommands.py", "chatcommunicate.py",
-    "chatexchange_extension.py", "datahandling.py", "deletionwatcher.py", "excepthook.py", "flovis.py",
-    "gitmanager.py", "globalvars.py", "helpers.py", "metasmoke.py", "nocrash.py", "parsing.py",
-    "spamhandling.py", "socketscience.py", "tasks.py", "ws.py",
-
-    "classes/feedback.py", "_Git_Windows.py", "classes/__init__.py", "classes/_Post.py",
+    "apigetpost.py",
```

Comment:
Seems like this should perhaps be a separate PR but +1

Reply:
There should be a separate PR. :; Ironically, I have a branch in process which rewrites this to invert the logic. Instead of a list of files to cause a full reboot, it uses a list of files which allow for just reloading the blacklist, or for which SD will complain into chat that the instance needs operator attention (e.g. nocrash.py, requirements.txt, and user_requirements.txt). In other words, IMO, the default for a change should be that there's a full reboot, rather than the default being to just reload the black/watchlists.  While that code is basically done, it needs testing and was definitely out of the scope of this PR. :)

For the inciting change here of adding entries for the new "editwatcher.py" and "recently_scanned_posts.py" files, I'd argue those should be should be in this PR, as this PR is the one adding those files. Given that I was making that change, and had recently encountered a time SD didn't do a full reboot when it should have when another .py file was changed, going through and adding the other missing Python files seemed the right thing to do. It would be quite reasonable to have the fixing up the list to the currently existing files be a separate commit, potentially in a different PR or direct commit, and have only adding the new "editwatcher.py" and "recently_scanned_posts.py" files in this PR. Admittedly, that's how I'd normally do it. I did choose to cut a corner here.